src/NServiceBus.TransportTests/.editorconfig

```diff
@@ -2,4 +2,7 @@
 
 # Justification: Test project
 dotnet_diagnostic.CA2007.severity = none
-dotnet_diagnostic.IDE0074.severity = none
\ No newline at end of file
+dotnet_diagnostic.IDE0074.severity = suggestion
+
+# IDE0028: Simplify collection initialization
```

Comment:
**Nitpick**: Can you please update this comment to have the same format as the others, meaning: "Justification:..."